tests/resources/promise-rejection-events.js

```diff
@@ -446,14 +446,14 @@ async_test(function(t) {
    'all inside a setTimeout');
 
 //
-// Positive unhandledrejection/rejectionhandled tests with delayed attachment
+// Negative unhandledrejection/rejectionhandled tests with task-delayed attachment
 //
 
 async_test(function(t) {
   var e = new Error();
   var p;
 
-  onUnhandledSucceed(t, e, function() { return p; });
+  onUnhandledFail(t, function() { return p; });
```

Comment:
Right, the task already exists, but the list may have been modified since the task was posted: "If the about-to-be-notified rejected promises list contains promise, remove promise from the about-to-be-notified rejected promises list and return."